spec/features/cli_vaccination_records_sync_spec.rb

```diff
@@ -6,6 +6,7 @@
   context "when the vaccination record exists and has not been synced" do
     it "syncs the vaccination record to the NHS API" do
       given_a_vaccination_record_exists
+      and_the_immunisations_fhir_api_integration_is_enabled
```

Comment:
Would it be cleaner and simpler to add the following after the `describe` block:

```
before do
  Flipper.enable :immunisations_fhir_api_integration
end

after do
  Flipper.disable :immunisations_fhir_api_integration
end
```

and then override it where you want to explicitly disable it (the innermost `before` block should be executed last):
```
before do
  Flipper.disable :immunisations_fhir_api_integration
end
```
and that way you can skip all the extra steps